openadapt/config.py

```diff
@@ -104,9 +104,24 @@ def getenv_fallback(var_name):
 DB_URL = f"sqlite:///{DB_FPATH}"
 DIRNAME_PERFORMANCE_PLOTS = "performance"
 
+
+def obfuscate(val, pct_reveal=0.1, char="*"):
+    num_reveal = int(len(val) * pct_reveal)
+    num_obfuscate = len(val) - num_reveal
+    obfuscated = char * num_obfuscate
+    revealed = val[-num_reveal:]
+    rval = f"{obfuscated}{revealed}"
+    assert len(rval) == len(val), (val, rval)
+    return rval
+
+
+
```

Comment:
Should  we remove extra new line here? As we should have a 2 newlines after a function. Here 3 were found. ?